src/settings/OrderTemplates/OrderTemplateView/OrderTemplateView.js

```diff
@@ -21,7 +21,7 @@ import { FundDistributionView } from '@folio/stripes-acq-components';
 
 import { PODetailsView } from '../../../components/PurchaseOrder/PODetails';
 import { SummaryView } from '../../../components/PurchaseOrder/Summary';
-import { RenewalsView } from '../../../components/PurchaseOrder/renewals';
+import { OngoingOrderInfoView } from '../../../components/PurchaseOrder/ongoingOgderInfo';
```

Comment:
should order template form be updated?

Reply:
I think, it shouldn't

Reply:
@uladislausamets if you have good arguments, it's better to provide them. From my perspective order templates should be aligned as well.